ci/test/00_setup_env_native_nowallet_libbitcoinkernel.sh

```diff
@@ -7,8 +7,8 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_nowallet_libbitcoinkernel
-export CI_IMAGE_NAME_TAG="docker.io/debian:bullseye"
-# Use minimum supported python3.9 and clang-16, see doc/dependencies.md
+export CI_IMAGE_NAME_TAG="docker.io/ubuntu:24.04"
```

Comment:
It's not the same change?

Reply:
It'd be good if the PR description explained. Rather than claiming to port changes but doing something different for unclear reasons.

Reply:
> It'd be good if the PR description explained. Rather than claiming to port changes but doing something different for unclear reasons.

My apologies. The PR description has been updated.